### PR TITLE
updated the osd_memory_target_cgroup_limit_ratio to 0.8 wrt bug 2012082

### PR DESCRIPTION
**Pull request by shivamdurgbuns**
updated the osd_memory_target_cgroup_limit_ratio to 0.8 wrt bug 2012082 in ocs_ci/ocs/constants.py file

### CODE REVIEW

**Review by keesturam — APPROVED** [review]
LGTM, please test the PR.

**Review by Avilir — APPROVED** [review]
LGTM

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -1504,7 +1517,7 @@
 
 # Values from configmap noobaa-postgres-config
 NOOBAA_POSTGRES_TUNING_VALUES = """
-max_connections = 300
+max_connections = 600
```
Comment (keesturam) [inline ocs_ci/ocs/constants.py:1520]: This will need changes based on the version too. i.e., until 4.9, it would be 300 and from 4.10, it would be 600.

Comment (shivamdurgbuns) [reply ocs_ci/ocs/constants.py:1520]: Ohh okay, will do it.

**Comment by shivamdurgbuns** [comment]
@keesturam Looks like the job passed.

**Review by keesturam — DISMISSED** [review]
Thanks for sending this PR and making the requested changes @shivamdurgbuns

**Review by keesturam — APPROVED** [review]

**Review by ebenahar — APPROVED** [review]

**Review by prsurve — APPROVED** [review]

**Comment by shivamdurgbuns** [comment]
Signing off this PR
    
    Signed-off-by: Shivam Durgbuns <sdurgbun@redhat.com>

**Pull request merged**